certified-connectors/Tribal/apiDefinition.swagger.json

```diff
@@ -1803,9 +1802,8 @@
             "x-ms-summary": "If-Match",
             "description": "ETag added in the If-Match header",
             "in": "header",
-            "required": false,
-            "type": "string",
-            "x-ms-visibility": "advanced"
+            "required": true,
```

Comment:
@mszy-tribal The parameter has been changed from optional to required. can you please confirm that no users will be affected by this change?

Reply:
This won't effect users because the parameter is required in the REST api that it calls. All this does is makes it clearer that they have to fill it in.